packages/contracts-versions/README.md

```diff
@@ -16,15 +16,35 @@ yarn add @aragon/osx-versions
 
 ## Usage
 
+Import Active contracts:
+
+```javascript
+// import active contracts from a specific version
+import {v1_2_0_active_contracts} from '@aragon/osx-versions';
+
+const mumbaiActiveContracts = v1_2_0_active_contracts.mumbai;
+```
+
+Import a specific contract source code from a specific version:
+
+```javascript
+
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity 0.8.17;
+
+// import legacy contracts from a specific version
+import {DAO} from '@aragon/osx-versions/versions/v1_0_1/contracts/core/dao/DAO.sol';
+
+.....
+```
+
+### Generate typechain
+
+To generate TypeChain if needed:
+
 ```javascript
-// import specific version
-import {v0_7_0_alpha_active_contracts, v0_7_0_alpha_typechain} from '@aragon/osx-versions';
-
-const typechain = v0_7_0_alpha_typechain;
-const idao: v0_7_0_alpha_typechain.IDAO = typechain.IDAO__factory.connect(
-  ethers.constants.AddressZero,
-  ethers.providers.getDefaultProvider()
-);
+find <path-to>/artifacts/@aragon/osx-versions/versions/ -name '*.json' -type f | grep -v '.dbg.json' | xargs typechain --target=ethers-v5 --out-dir <path-to>/typechain/osx-versions/versions/"
```

Comment:
Could there be an (npm) script to do this?

Reply:
it is just a recommendation for consumers of this package 
if they want they can do something similar to this command

Reply:
No